python/examples/feature-selection.py

```diff
@@ -0,0 +1,116 @@
+import sys
+
+import chemfiles
+import numpy as np
+from equistore import Labels, TensorBlock, TensorMap
+from skcosmo.feature_selection import FPS
+
+from rascaline import SoapPowerSpectrum
+
+
+with chemfiles.Trajectory(sys.argv[1]) as trajectory:
+    frames = [f for f in trajectory]
+
+HYPER_PARAMETERS = {
+    "cutoff": 5.0,
+    "max_radial": 6,
+    "max_angular": 4,
+    "atomic_gaussian_width": 0.3,
+    "center_atom_weight": 1.0,
+    "radial_basis": {
+        "Gto": {},
+    },
+    "cutoff_function": {
+        "ShiftedCosine": {"width": 0.5},
+    },
+}
+
+calculator = SoapPowerSpectrum(**HYPER_PARAMETERS)
+
+descriptor = calculator.compute(frames)
+
+# The selections for feature can be a set of `Labels`, in which case
+# the names of the labels must be a subset of the names of the
+# properties produced by the calculator. You can see the default set
+# of names with:
+print("properties names:", descriptor.property_names)
+
+# Now we can use these names for selection.
+# In this case, only features  matching the labels in this selection
+# will be used
+selection = Labels(
+    names=["l"],
+    values=np.array([[0]]),
+)
+reference_descriptor = calculator.compute(frames, selected_properties=selection)
+
+# The previous selection method uses the same selection for all blocks.
+# If you can to use different selection for different blocks, you should
+# use a `TensorMap` to create your selection
```

Comment:
This comment should move after the print code below since it refers to stuff happening later